rq_scheduler/scheduler.py

```diff
@@ -188,21 +214,34 @@ def enqueue_at(self, scheduled_time, func, *args, **kwargs):
         scheduler = Scheduler(queue_name='default', connection=redis)
         scheduler.enqueue_at(datetime(2020, 1, 1), func, 'argument', keyword='argument')
         """
-        timeout = kwargs.pop('timeout', None)
-        job_id = kwargs.pop('job_id', None)
-        job_ttl = kwargs.pop('job_ttl', None)
-        job_result_ttl = kwargs.pop('job_result_ttl', None)
-        job_description = kwargs.pop('job_description', None)
-        depends_on = kwargs.pop('depends_on', None)
-        meta = kwargs.pop('meta', None)
-        queue_name = kwargs.pop('queue_name', None)
-        on_success = kwargs.pop('on_success', None)
-        on_failure = kwargs.pop('on_failure', None)
-
-        job = self._create_job(func, args=args, kwargs=kwargs, timeout=timeout,
-                               id=job_id, result_ttl=job_result_ttl, ttl=job_ttl,
-                               description=job_description, meta=meta, queue_name=queue_name, depends_on=depends_on,
-                               on_success=on_success, on_failure=on_failure)
+        timeout = kwargs.pop("timeout", None)
+        job_id = kwargs.pop("job_id", None)
+        job_ttl = kwargs.pop("job_ttl", DEFAULT_TTL)
```

Comment:
What do you think about creating 3 independent vars and env vars for them with the same value?